hera_cal/io.py

```diff
@@ -716,39 +716,40 @@ def load_flags(flagfile, filetype='h5', return_meta=False):
         return flags
 
 
-def get_file_lst_range(filepaths, filetype='uvh5', add_int_buffer=False):
+def get_file_times(filepaths, filetype='uvh5'):
     """
-    Get a file's start, stop and dlst in radians.
+    Get a file's lst_array in radians and time_array in Julian Date.
 
-    Start and stop are bin center. Miriad standard is bin start,
-    so shift by int_time / 2 is done. UVH5 standard is bin center,
-    so lsts are left untouched.
+    Miriad standard is bin start, so a shift by int_time / 2 is performed.
+    UVH5 standard is bin center, so times are left untouched.
+
+    Note: this is not currently compatible with Baseline Dependent Averaged data.
 
     Args:
         filepaths : type=list, list of filepaths
         filetype : str, options=['miriad', 'uvh5']
-        add_int_buffer : type=bool, if True, extend stop times by an
-            integration duration. This is done so that when LST matching
-            the last bin is appropriately accounted for.
 
     Returns:
         If input is a string, output are floats, otherwise outputs are lists.
```

Comment:
This isn't quite true... the third and fourth outputs are numpy arrays or lists of numpy arrays.

Reply:
Also, the outputs are numpy arrays, not lists.